demo/index.html

```diff
@@ -17,25 +17,22 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
 
-    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
 
-    <script type="module" src="../../polymer/polymer-legacy.js"></script>
-    <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-    <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-    <script type="module" src="../iron-image.js"></script>
-
-    <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
     <script type="module">
-const $_documentContainer = document.createElement('template');
+      import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+      import '@polymer/iron-demo-helpers/demo-snippet.js';
+      import '@polymer/polymer/polymer-legacy.js';
+      import '../iron-image.js';
+
+      window.load = function(id) {
+        document.getElementById(id).src = './polymer.svg?' + Math.random();
+      };
```

Comment:
`type=module` is deferred, so I'm not sure setting `window.load` works (just check, no action needed if it works).

Reply:
This function is called by some inline click listeners on buttons in the page below. I think it'll be ok as long as it runs before the user tries to click them.